.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Github Actions
+
+on: [push]
```

Comment:
can you add `pull_request` there (maybe that's why it didn't trigger yet)

Reply:
okay i'll add that too